rascal-vscode-extension/src/test/vscode-suite/dsl.test.ts

```diff
@@ -103,14 +110,10 @@ parameterizedDescribe(function (errorRecovery: boolean) {
         await fs.writeFile(TestWorkspace.picoFile, picoFileBackup);
     });
 
-    it("have highlighting and parse errors", async function () {
+    it("has highlighting and parse errors", async function () {
         await ignoreFails(new Workbench().getEditorView().closeAllEditors());
         const editor = await ide.openModule(TestWorkspace.picoFile);
-        const isPicoLoading = ide.statusContains("Pico");
-        // we might miss this event, but we wait for it to show up
-        await ignoreFails(driver.wait(isPicoLoading, Delays.normal, "Pico parser generator should have started"));
-        // now wait for the Pico parser generator to disappear
-        await driver.wait(async () => !(await isPicoLoading()), Delays.verySlow, "Pico parser generator should have finished", 100);
+        await waitForPico();
```

Comment:
this waiting should only be in the first test of the file.

So I don't think we should be extracting this into a separate function and calling again a bit later, as that would be hiding a different bug.

(the whole reason it's a copy of `loadPico` is that sometimes we've had issues where the original loader missed a step in waiting, and then while we're already looking for syntax errors, the VS Code extension on a slow CI server is still loading the language).